config/Directory.Build.props

```diff
@@ -1,12 +1,12 @@
+<!-- This file was generated by /configure.ps1, do not edit directly. -->
 <Project>
   <PropertyGroup>
-    <SteeltoeVersion>3.2.*</SteeltoeVersion>
+    <SteeltoeVersion>3.2.*-*</SteeltoeVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <GitInfoVersion>2.0.20</GitInfoVersion>
-    <JsonNetVersion>11.0.3</JsonNetVersion>
```

Comment:
Are any of the samples not touched in this PR still using `JsonNetVersion`?

Reply:
I haven't found any occurrences of `$(JsonNetVersion)` in the samples repository.

However, there are a few usages of `$(GitInfoVersion)`, all from within MusicStore (which appears to be [abandoned](https://github.com/aspnet/MusicStore)). Would it be okay to inline the version there, or do you think it needs to stay in `Directory.Build.props`?